codalab/model/bundle_model.py

```diff
@@ -2901,3 +2903,112 @@ def delete_oauth2_auth_code(self, auth_code_id):
             connection.execute(
                 oauth2_auth_code.delete().where(oauth2_auth_code.c.id == auth_code_id)
             )
+
+    # ===========================================================================
+    # Bundle Store methods follow!
+    # ===========================================================================
+    def get_bundle_stores(self, user):
+        with self.engine.begin() as connection:
+            rows = connection.execute(
+                select(
+                    [
+                        cl_bundle_store.c.uuid,
+                        cl_bundle_store.c.owner_id,
+                        cl_bundle_store.c.name,
+                        cl_bundle_store.c.storage_type,
+                        cl_bundle_store.c.storage_format,
+                        cl_bundle_store.c.url,
+                    ]
+                ).where(
+                    or_(
+                        cl_bundle_store.c.owner_id == 'codalab(0)',
```

Comment:
Document this in a comment that it returns all bundles owned by the root user or current user